client/verta/tests/test_cli/test_registry.py

```diff
@@ -52,8 +52,37 @@ def test_get_model_wrong_name_error(self, strs):
 
 
 class TestList:
-    pass
+    def test_list_model(self):
+        client = Client()
+        model1 = client.get_or_create_registered_model()
+        label = model1._msg.name + "label1"
+        model1.add_label(label)
+        model1.add_label("label2")
+        client.get_or_create_registered_model()
+        model = client.get_or_create_registered_model()
+        model.add_label(label)
+        runner = CliRunner()
+        result = runner.invoke(
+            cli,
+            ['registry', 'list', 'registeredmodel', '--filter', "labels == \"{}\"".format(label)],
+        )
+
+        assert not result.exception
+        assert 'result count: 2' in result.output
+        assert str(model1._msg.name) in result.output
+        assert str(model._msg.name) in result.output
+
+        result = runner.invoke(
+            cli,
+            ['registry', 'list', 'registeredmodel', '--filter', "labels == \"{}\"".format(label), "--output=json"],
+        )
+
+        assert not result.exception
+        assert 'result count: 2' in result.output
+        assert str(model1._msg.name) in result.output
+        assert str(model._msg.name) in result.output
```

Comment:
Could you also add one more test case that does _not_ use `--filter`?

Reply:
yes, I added it now. But it lists all the models that I have created before (works ~ 1 minute). Are you sure that it is needed? When you run these tests using a test account can you delete these unnecessary entities?

Reply:
Hmm you are right that it takes more time to run, but I would like to make sure the command works properly even when the user does not pass `--filter`.

To reduce execution time, we could maybe remove `--filter` from the `--output=json` case.

I will make another PR to make sure test entities are deleted!

Reply:
moved this case to --filter